templates/python/inner.html

```diff
@@ -623,11 +623,11 @@ <h1 id="form">Form Example</h1>
 
 {% block left_sidebar %}
 <aside class="left-sidebar" role="secondary">
-  
+
     {% include "components/navigation-widget.html" %}
 
     <div class="twitter-widget sidebar-widget">
-        <script type="text/javascript" src="http://widgets.twimg.com/j/2/widget.js"></script>
+        <script type="text/javascript" src="https://widgets.twimg.com/j/2/widget.js"></script>
```

Comment:
It would better to use the same style in both this file and `templates/waitforit.html` (`//...` was used instead of `https://...` there)